### PR TITLE
VET TEC: Direct deposit information page enhancements #17372

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

As a Veteran I need to understand the results of updating my direct deposit information within the VET TEC workflow.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/17372

## Testing done
- [ ] local testing
- [ ] unit testing

## Screenshots
Screen Shot 2019-04-04 at 1 11 03 PM
Screen Shot 2019-04-04 at 1 11 33 PM

## Acceptance criteria
- [ ] 1. Rename button from "New Account" to "Update Account"
- [ ] 2. Dynamically update the message that appears on this page so that if a Veteran has bank info (prefilled) they see one message, and if they do not have bank info then they see a different message
     - [ ] Message to display when prefill exists: "This is the bank account information we have on file for you. We’ll pay your housing stipend to this account."
     - [ ] Message to display when prefill does not exist: "We don’t have your bank information on file. Please provide your direct deposit information below. We’ll pay your housing stipend to this account."
- [ ] 3. When the applicant clicks the "Save" button after editing their direct deposit information, the following message is displayed: “Thank you for providing your direct deposit information. This new account won’t be updated right away. We'll deposit your housing stipend into this account if your VET TEC application is approved.”  (See Screenshot for example)
     - [ ] The left border will have the green success color.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/components/PaymentView.jsx [hunk 1]
```diff
@@ -40,6 +50,9 @@ export const PaymentView = ({ formData = {}, originalData = {} }) => {
         <p>Account number: {maskBankInformation(bankAccountNumber, 4)}</p>
         <p>Bank routing number: {maskBankInformation(bankRoutingNumber, 4)}</p>
       </div>
+      {hasNewBankAccountInfo && (
+        <AlertBox isVisible status="success" content={directDepositAlert} />
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/components/PaymentView.jsx]: Similar to another PR earlier, you could pass `hasNewBankAccountInfo` to the `isVisible` prop and remove the conditional.

**Review by jbalboni — APPROVED** [review]

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
Covers changes for https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/17830

**Review by jbalboni — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]
LGTM

**Pull request merged**